ext/CatalystHomotopyContinuationExtension/homotopy_continuation_extension.jl

```diff
@@ -36,7 +36,7 @@ Notes:
 ```
   """
 function Catalyst.hc_steady_states(rs::ReactionSystem, ps; filter_negative = true,
-        neg_thres = -1e-20, u0 = [], kwargs...)
+        neg_thres = -1e-15, u0 = [], kwargs...)
```

Comment:
On the latest Julai version, the accuracy of HC.jl seems slightly affected. I.e. before values that should be zero could be negative, but _incredibly_ small. Now I have seen values around -1e-17 (which caused test failures). I will tell Paul at HC.jl about it, just in case he is interested,